### PR TITLE
Icon

### PR DESCRIPTION
**Pull request by HarshitAditya27**
## What is the change?
I have added a favicon for the website, code written in header.ejs (code:).The favicon is located in the images folder which is inside the public folder and I have set the value of const URL which is const URL = process.env.URL; (as instructed by you) and one more thing the favicon created is simple it's just the logo with the border of color combination white and pink. I hope you will like my favicon :). (This PR is the updated version of issue no #107 )

Related issue?

## Related issue?
issue: #105  

## How was it tested?
I have tested the website by using the hyper terminal and mongod server.

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [x] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:
Add a screenshot or demo video if appropriate. 
Screenshot (574)_LI

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -40,14 +40,16 @@ app.set('view engine', 'ejs');
 app.use(bodyParser.urlencoded({extended: true}));
 app.use(express.static("public"));
 
+
+
```
Comment (Pranav016) [inline app.js:44]: Why were these extra lines added

File: app.js [hunk 2]
```diff
@@ -40,14 +40,16 @@ app.set('view engine', 'ejs');
...
 //When in development mode then only require the dotenv module
 if(process.env.NODE_ENV !== 'production'){
   const dotenv = require('dotenv');
   dotenv.config({path:'./.env'});
 }
 
 //Connecting to Mongo Database using ODM Mongoose-
-const URL = process.env.URL;
+ const URL = process.env.URL;
```
Comment (Pranav016) [inline app.js:52]: what was the need of adding space before this

File: views/partials/header.ejs [hunk 3]
```diff
@@ -5,8 +5,10 @@
   <meta charset="utf-8">
   <title>Daily Journal</title>
   <meta charset="utf-8">
+
```
Comment (Pranav016) [inline views/partials/header.ejs:8]: extra line added

**Review by Pranav016 — CHANGES_REQUESTED** [review]
Delete the favicon from the assets folder if you've already added it to the images folder in public dir

**Comment by Pranav016** [comment]
There are two commits in this PR, both for adding a favicon. Please avoid unnecessary commits, commit history needs to be clean. (Follow Contribution Guidelines)

File: app.js (hunk 1 above)
Comment (HarshitAditya27) [reply app.js:44]: Sir, I was not knowing about that maybe by mistake I added those lines please forgive me.

Comment (Pranav016) [reply app.js:44]: It's fine, just the work in 1 commit. Make a branch from master and then work on it or squash your commits and then force push it.

Comment (HarshitAditya27) [reply app.js:44]: Sir I  have deleted the two extra lines and removed the favicon from the assets folder, Are there any more changes that I have to do?

File: app.js (hunk 2 above)
Comment (HarshitAditya27) [reply app.js:52]: Sir the file which I have pushed does not contain two const URLs,I don't know why it is showing a two const URL over here?

File: app.js (hunk 1 above)
Comment (Pranav016) [reply app.js:44]: No

File: app.js (hunk 2 above)
Comment (Pranav016) [reply app.js:52]: This called diff, its not two URLs

File: app.js (hunk 1 above)
Comment (HarshitAditya27) [reply app.js]: Excuse me, sir, I have deleted extra lines and favicon from the assets folder as you have requested, can you see my commits? 
I hope I have resolved your issue :)

File: app.js (hunk 2 above)
Comment (HarshitAditya27) [reply app.js:50]: ooh ok

**Comment by Pranav016** [comment]
There are 4 commits just for adding the favicon, please squash commits or make a PR from a new branch. The commit messages are incorrect.
`Deleted Favicon from assets (as requested)`

**Comment by HarshitAditya27** [comment]
Sir, I have sent you a screenshot that contains the commit history of this PR, I have assigned numbers to each commit, sir can you tell me which commits I have to squash by giving me the number of commit that I have assigned so that it would be easy for me to which commit I have to squash.
Screenshot (575)_LI

**Comment by Pranav016** [comment]
> Sir, I have sent you a screenshot that contains the commit history of this PR, I have assigned numbers to each commit, sir can you tell me which commits I have to squash by giving me the number of commit that I have assigned so that it would be easy for me to which commit I have to squash.
> Screenshot (575)_LI

Squash 2,3,4 into 1

**Comment by HarshitAditya27** [comment]
Ok sir I will be squashing 2,3, and 4.

**Comment by HarshitAditya27** [comment]
Sir, I have squashed commit 2,3, and 4, please let me know if any error exists. Sir, I hope you like my changes :)

**Comment by HarshitAditya27** [comment]
Sir, I will be creating a new PR because it is becoming very hectic to squash the commits.

**Comment by Pranav016** [comment]
> Sir, I will be creating a new PR because it is becoming very hectic to squash the commits.

Yup, that'll be easier 👍

**Comment by HarshitAditya27** [comment]
Sir, I have already created a new PR  name "Added favicon successfully #117 " with only 1 commit and no extra lines (I have checked each and every document (which I have used)).I hope you will like my work :) and hope this time its gets merged ! .

**Pull request closed without merging**